app/models/translation.rb

```diff
@@ -15,9 +15,14 @@ class Translation < ActiveRecord::Base
   validates_with UsesOneskyValidator
 
   before_destroy :prevent_destroy_published, if: :is_published
-  before_update :push_published_to_s3
```

Comment:
I believe the legacy publishing process triggered this update from the `drafts_controller`, if that is the case we should update that logic to trigger `push_published_to_s3` from there if the request indicates `is_published: true`